packages/amplify_datastore/android/build.gradle

```diff
@@ -72,10 +72,10 @@ dependencies {
     // Support for Java 8 features
     coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:2.0.3'
 
-    implementation 'com.amplifyframework:aws-auth-cognito:2.10.0'
-    implementation "com.amplifyframework:aws-api:2.10.0"
-    implementation "com.amplifyframework:aws-datastore:2.10.0"
-    implementation "com.amplifyframework:aws-api-appsync:2.10.0"
+    implementation 'com.amplifyframework:aws-auth-cognito:2.11.0'
```

Comment:
fyi for reviewers - dependabot only bumped data store in this PR. I updated it to include all packages.